apps/remix-ide/src/app/tabs/test-tab.js

```diff
@@ -144,17 +144,40 @@ module.exports = class TestTab extends ViewPlugin {
       this.testsOutput.appendChild(this.outputHeader)
     } else if (result.type === 'testPass') {
       this.testsOutput.appendChild(yo`
-        <div id="${this.runningTestFileName}" data-id="testTabSolidityUnitTestsOutputheader" class="${css.testPass} ${css.testLog} text-success border-0">
+        <div id="${this.runningTestFileName}" data-id="testTabSolidityUnitTestsOutputheader" class="${css.testPass} ${css.testLog} bg-light mb-2 text-success border-0">
           ✓ ${result.value}
         </div>
       `)
     } else if (result.type === 'testFailure') {
-      this.testsOutput.appendChild(yo`
-        <div class="${css.testFailure} ${css.testLog} d-flex flex-column text-danger border-0" id="UTContext${result.context}">
+      if (!result.assertMethod) {
+        this.testsOutput.appendChild(yo`
+        <div class="bg-light mb-2 ${css.testFailure} ${css.testLog} d-flex flex-column text-danger border-0" id="UTContext${result.context}">
           <span> ✘ ${result.value}</span>
-          <span>"${result.errMsg}"</span>
+          <span class="text-dark">Error Message:</span>
+          <span class="pb-2 text-break">"${result.errMsg}"</span>
         </div>
       `)
+      } else {
+        const preposition = result.assertMethod === 'equal' || result.assertMethod === 'notEqual' ? 'to' : ''
+        const method = result.assertMethod === 'ok' ? '' : result.assertMethod
+        const expected = result.assertMethod === 'ok' ? `'true'` : result.expected
+        this.testsOutput.appendChild(yo`
+          <div class="bg-light mb-2 ${css.testFailure} ${css.testLog} d-flex flex-column text-danger border-0" id="UTContext${result.context}">
+            <span> ✘ ${result.value}</span>
+            <span class="text-dark">Error Message:</span>
+            <span class="pb-2 text-break">"${result.errMsg}"</span>
+            <span class="text-dark">Asserttion:</span>
+            <div class="d-flex flex-wrap">
+              <span>Expected value should be</span>
+              <div class="mx-1 font-weight-bold">${method}</div>
+              <div>${preposition} ${expected}</div>
+            </div>
+            <span class="text-dark">Received value:</span>
+            <span>${result.returned}</span>
+            <span class="text-dark text-sm pb-2">Skipping the rest...</span>
```

Comment:
`Skipping the rest...` is not so informative. I think `Skipping remaining tests` will provide better info.

Reply:
good point

Reply:
should it be skipping the remaining tests of a function? or without function?

Reply:
I think, we can update the text as : `Skipping the remaining tests of the function` => `Skipping the remaining tests...`
Because we already show tests corresponding to a function in a box

Reply:
@ioedeveloper I think this text has not been changed yet